### ISSUE TITLE
[feature request] React cards.

### ISSUE THREAD

**Issue by swider33** [db0/Doomtown-Reloaded-OCTGN#63]
Is it possible to implement yes/no menu for a player who controls a card with react ability to use that ability?

**Comment by db0** [comment]
I don't follow. You want an extra yes/no after you try to use a react? Or
automate the react completely and add a yes/no confirm?

**Comment by swider33** [comment]
I want a menu to pop up window asking if you want to use your react ability when appropriate: for instance, when you are creating a gadget and your home home is gadgetorium a pop up window would display saying if you want to use your react ability.

**Comment by db0** [comment]
Sorry but I won't do that. This would make the player have to swat away
pop-ups all the time, instead of just having to ctrl-z once when they want
to. Sometimes I will do this when there's no reason not to use the react
the first time, but for most reacts, this is not the case.

**Comment by swider33** [comment]
I understand, the main reason I asked for that is gadgetorium, since when people are using it it is so messy and unclear. I think the skill check isn't automatic, then they are using Specks  and on top of that gadgetorium. Especially when the screen is zoomed out it is hard to follow what is going on unless you know very well every card your opponent is using.

**Comment by db0** [comment]
I know, but the alternative is the gadgetorium pestering you every time you
make a skill check, unless you boot it. Also it would take too much effort
to code for marginal benefit :-/

**Comment by swider33** [comment]
Any chance to add automatic skill check on gadgets inventing? You answered me some time ago it was a feature not a bug ( inventing not being automated).

**Comment by db0** [comment]
There is an automatic skill check, but the gadget discarding is just not
automated on fail. This is still a feature as there's a lot of times
something might be missed (e.g. gadgetorium). So you just discard manually.

**Comment by swider33** [comment]
Ok, I think we can close the issue. Thanks for answering questions.
